manifests/w/Workday,Inc/PlanningforExcel/2022.105.2718.2929/Workday,Inc.PlanningforExcel.locale.en-US.yaml

```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: Workday,Inc.PlanningforExcel
+PackageVersion: 2022.105.2718.2929
+PackageLocale: en-US
+Publisher: Workday, Inc.
+PackageName: Planning for Excel
+License: Workday Inc.
+Copyright: Copyright (c) Workday, Inc.. All rights reserved.
+ShortDescription: Planning for Excel

```

Comment:
Please provide a better short description of what the application is used for

Reply:
Workday Planning for Excel enables companies of all sizes to collaboratively plan and model, easily access real-time analytics and streamline complex reporting.